e2e-tests/content-strings/pages/insurance/declarations/anti-bribery-code-of-conduct/index.js

```diff
@@ -1,23 +1,35 @@
 import { SHARED } from '../shared';
 import { LINKS } from '../../../../links';
 
+const HINT = {
+  INTRO: 'If no, your attention is drawn to',
+  LINK: {
+    TEXT: 'Ministry of Justice guidance',
+    HREF: LINKS.EXTERNAL.BRIBERY_ACT_2010_GUIDANCE,
+  },
+};
+
+const ANSWER_YES_REVEAL = {
+  TEXT: "We will email you after you submit your application (also known as a 'proposal')  to request your anti-bribery code of conduct.",
+};
+
 export const ANTI_BRIBERY_CODE_OF_CONDUCT = {
   VERSIONS: [
     {
-      VERSION: 1,
+      VERSION: '1',
```

Comment:
Do the version needs to be a string?

Reply:
yes. all versions are strings incase we have 1.1 for example